tests/trainers/test_so2sat.py

```diff
@@ -44,6 +44,7 @@ def config(self, request: SubRequest, bands: Tuple[str, int]) -> Dict[str, Any]:
         task_args["in_channels"] = bands[1]
         loss, weights = request.param
         task_args["loss"] = loss
+        task_args["num_classes"] = 17
```

Comment:
Is this needed now that the default configs have been updated?

Reply:
Good question, it's not. I'll remove. Can you add it as a suggestion so I can do it from my phone lol.

Reply:
I'm also on my phone lol